src/main/java/ghbot/GhBot.java

```diff
@@ -0,0 +1,56 @@
+package ghbot;
+
+import java.io.IOException;
+import java.time.format.DateTimeParseException;
+
+import ghbot.exception.GhBotException;
+import ghbot.executor.ByeExecutor;
+import ghbot.executor.Executor;
+import ghbot.parser.Parser;
+import ghbot.storage.Storage;
+import ghbot.task.TaskList;
+import ghbot.ui.Ui;
+
+/**
+ * GhBot Class.
+ * This is the main class for the chatbot.
+ */
+public class GhBot {
+    private Storage storage;
+    private TaskList taskLst;
```

Comment:
typo? I like that the variables are written in camelCase but perhaps could rename as taskList.